dev_process.md

```diff
@@ -0,0 +1,144 @@
+AIRR Standards Development Process
+----------------------------------
+
+The airr-standards repository contains numerous sub-projects within it
+which are all highly interconnected. Our goal is a simple workflow for
+development that also insures stable versioning and releases for AIRR
+data and software associated with publications and database
+submissions. The current sub-projects include:
+
+* Minimum information about an Adaptive Immune Receptor Repertoire Sequencing Experiment (MiAIRR).
+
+* Specification of file formats:
+
+1. Alignment hit table
+
+2. Rearrangement annotations
+
+* OpenAPI definitions:
+
+1. MiAIRR objects
+
+2. Rearrangement annotations
+
+* Documentation for http://docs.airr-community.org
+
+* Python package for reading/writing file formats
+
+Some future sub-projects include:
+
+* Ontologies definitions for AIRR data elements.
+
+* Common Repository WG defining standard API with associated specification.
+
+* R package for file format reading/writing/validation.
+
+* Germline Database WG defining criteria, submission protocols, and
+  tools, with associated file formats and specifications.
+
+* Software WG defining standardized data sets and metrics.
+
+### Git Structure
+
+The `master` branch is utilized for all development for the ongoing
+major version of the AIRR Standards. The versioning system, described
+in separate section below, is designed to allow parallel branches of
+development and maintenance for multiple AIRR Standards, though
+currently there is a single version available, MiAIRR V1. When
+multiple major versions of the AIRR Standards are available, then
+these instructions will be updated to reflect the development process
+for older versions.
+
+* Commits to `master` can only be performed by the core team, which
+  currently includes Syed Ahmad Chan Bukhari, Christian Busse, Scott
+  Christley, Jason Vander Heiden, and Uri Laserson. Even so, consider
+  using a pull request if the changes affect important MiAIRR or
+  specifications files that may impact others. The core team will be
+  expanded as the number of sub-projects in airr-standards grows.
```

Comment:
I would add an explicit note that the guideline is "don't break stuff for other people".  And also explicitly call out as an example that "renaming a field" breaks stuff and would require review by others.